Example/GiniVision/AppDelegate.swift

```diff
@@ -58,17 +58,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         do {
             try document?.validate()
             
-            // 4. Create alert which allows user to open imported file either with ScreenAPI or ComponentAPI
+            // 4. Create an alert which allow users to open imported file either with the ScreenAPI or the ComponentAPI
             alertViewController = UIAlertController(title: "Importierte Datei", message: "Möchten Sie die importierte Datei mit dem ScreenAPI oder ComponentAPI verwenden?", preferredStyle: .alert)
             
             alertViewController.addAction(UIAlertAction(title: "Screen API", style: .default) { _ in
-                screenAPIVC.present(screenAPIVC.giniScreenAPI(withImportedDocument: document), animated: true, completion: nil)
+                selectAPIVC.present(selectAPIVC.giniScreenAPI(withImportedDocument: document), animated: true, completion: nil)
             })
             
             alertViewController.addAction(UIAlertAction(title: "Component API", style: .default) { _ in
-                if let componentAPI = screenAPIVC.giniComponentAPI(withImportedDocument: document) {
-                    screenAPIVC.present(componentAPI, animated: true, completion: nil)
-                }
+//                if let componentAPI = selectAPIVC.giniComponentAPI(withImportedDocument: document) {
```

Comment:
I suppose this will never be uncommented. Just remove it.

Reply:
It will never be. Removed!